### ISSUE TITLE
Prompt for sudo authorization before building

### ISSUE THREAD

**Issue by stereomato** [fosskers/aura#944]
Paru does this, iirc. Sometimes I do other stuff with my computer when I leave aura building a package, and as such I always miss the prompt for sudo, only realizing when the Console app gives me a "aura command has finished" notification.

**Comment by fosskers** [comment]
`sudo` isn't necessary for building, only the final installation, in which it does prompt you for `sudo`. It sounds like the `sudo` request is timing out?

**Comment by stereomato** [comment]
Yeah, that. Sorry, I got confused in that bit yeah. It times out when asking for installation yes.

**Comment by RubenKelevra** [comment]
> `sudo` isn't necessary for building, only the final installation, in which it does prompt you for `sudo`. It sounds like the `sudo` request is timing out?

Yeah, building complete, user deep at sleep, aura timing out ;)

``` 
[sudo] password for ruben: 
sudo: timed out reading password
sudo: a password is required
aura :: A call to pacman -U failed.
```

Another workaround would be to not fail hard if that happens, but just to wait for an "Enter" by the user to show this prompt again.

However, paru has a nice convenient feature, where it loops a sudo as long as it's building, so it retains the ability to run sudo without a password prompt.

**Comment by stereomato** [comment]
yeah!
